src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -658,26 +658,26 @@ static void window_cheats_money_mousedown(rct_window* w, rct_widgetindex widgetI
             break;
         case WIDX_MONTH_UP:
             _monthSpinnerValue++;
-            _monthSpinnerValue = std::clamp(_monthSpinnerValue, 1, (int)MONTH_COUNT);
-            _daySpinnerValue = std::clamp(_daySpinnerValue, 1, (int)days_in_month[_monthSpinnerValue - 1]);
+            _monthSpinnerValue = std::clamp(_monthSpinnerValue, 1, static_cast<int>(MONTH_COUNT));
```

Comment:
this should be using `int32_t` instead of `int`. Same goes for the other clamps.